Orange/widgets/tests/base.py

```diff
@@ -1,11 +1,13 @@
+import os
 import unittest
-import sip
+
 
 import numpy as np
 from AnyQt.QtWidgets import (
     QApplication, QComboBox, QSpinBox, QDoubleSpinBox, QSlider
 )
 
+import Orange.data
```

Comment:
Table is already imported in the line 15. Maybe the rest of the symbols could be imported in the same fashion?